src/Kris/LaravelFormBuilder/Form.php

```diff
@@ -618,6 +630,19 @@ public function __get($name)
     }
 
     /**
+     * Set the Event Dispatcher to fire Laravel events
+     *
+     * @param EventDispatcher $eventDispatcher
+     * @return $this
+     */
+    public function setEventDispatcher(EventDispatcher $eventDispatcher)
```

Comment:
We can remove this setter, and pull eventDispatcher form formHelper (add getter there)